client/src/App.js

```diff
@@ -11,6 +12,12 @@ function App() {
       <Main>
         <Home />
         <ErrorCard title="Error" message="message" />
+        <MovieCard
+          title="Fight Club"
+          year="1999"
+          genre="Drama"
+          imgUrl="https://image.tmdb.org/t/p/w500/a26cQPRhJPX6GbWfQbvZdrrp9j9.jpg"
```

Comment:
How about working with mock data for now? That way you are more or less done with implementing and mapping over data and don't have to add that later.

Reply:
This comes with the next pr, makes more sense to implement it there.